src/rgw/driver/rados/rgw_rest_log.cc

```diff
@@ -703,6 +713,7 @@ void RGWOp_DATALog_List::send_response() {
 
   s->formatter->open_object_section("log_entries");
   s->formatter->dump_string("marker", last_marker);
+  //s->formatter->dump_float("last_updated", last_update);
```

Comment:
please return it this way, instead of adding a timestamp to every `rgw_data_change_log_entry`

you can use `encode_json("last_updated", last_update, s->formatter)` to encode the `ceph::real_time` variable as-is

Reply:
Where the response for this op (or these admin ops in general) are being read was something I couldn't understand reading through the code. Could you show me an example on the code?

Reply:
the values encoded here are decoded using the `read_remote_data_log_response` type in `RGWReadRemoteDataLogShardCR`:

https://github.com/ceph/ceph/blob/a4f58d336960c79b2dd69b7188e930cb497bfaf3/src/rgw/driver/rados/rgw_data_sync.cc#L299C8-L380